dashboards/proposal.md

```diff
@@ -0,0 +1,74 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Dynamic Dashboards from Notebooks (jupyter-incubator/dashboards)
+
+## Development team and Advocate
+
+Subproject development team:
+
+* Peter Parente, IBM (`@parente`)
+* Gino Bustelo, IBM (`@ginobustelo`)
+* Justin Tyberg, IBM (`@jtyberg `)
+* Dan Gisolfi, IBM (`@vinomaster`)
+
+Steering Council Advocate:
+
+* Brian Granger (`@ellisonbg`)
+
+## Subproject goals, scope and functionality
+
+### Concept 
+
+>What if the notebook author could quickly and easily publish rich interactive dashboard applications that allow users to perform interactive visual data exploration? 
+
+Today, notebook authors can share their notebooks for others to view and run in the Jupyter Notebook web application. Authors can also transform their notebooks into a variety of static formats for ease-of-viewing in common tools like web browsers or PDF readers. However, if the notebook author desired to generate a dashboard application that would interact with the insights derived in the notebook, she would have to step outside the authoring environment to build a traditional web application. Such a task involves a great deal of copy/pasting and rewriting code that already exists in the notebook.
+
+This incubation proposal pertains to a research sandbox associated with investigating ways of transforming notebooks directly into dashboards. The proposed repo would be based on a proof-of-concept extension for Jupyter Notebook that builds upon existing community work like [Thebe](https://github.com/oreillymedia/thebe) and the [recent refactoring of the Jupyter Notebook](http://blog.jupyter.org/2015/04/15/the-big-split/) in order to demonstrate the concept of producing dynamic dashboards from notebooks.
+
+### Goals
+
+* Provide within-notebook enhancements for laying out notebook cell outputs in a dashboard format, capturing layout metadata in the notebook document, and converting notebook documents into deployable web apps.
```

Comment:
I am +1 on this, but I think a key part of the incubation period will be integrating with our existing JS refactoring efforts to build tabbed/paneled layout in the notebook app.